cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -568,6 +568,7 @@ func (s *Server) Index(args *indexArgs) (state indexState, err error) {
 		}
 	}
 
+	log.Printf("updating index %d", len(args.LanguageMap))
```

Comment:
is this useful to keep for now? if so rather than an extra output line can you add it into the log line on the next line?

Reply:
Oops meant to remove this  😅